test/minio/1.2.0/templates/_helpers.tpl

```diff
@@ -0,0 +1,40 @@
+{{/*
+Determine secret name.
+*/}}
+{{- define "minio.secretName" -}}
+{{- include "common.names.fullname" . -}}
+{{- end -}}
+
+
+{{/*
+Retrieve true/false if minio certificate is configured
+*/}}
+{{- define "minio.certAvailable" -}}
+{{- if .Values.certificate -}}
+{{- $values := (. | mustDeepCopy) -}}
+{{- $_ := set $values "commonCertOptions" (dict "certKeyName" $values.Values.certificate) -}}
+{{- template "common.resources.cert_present" $values -}}
+{{- else -}}
+{{- false -}}
+{{- end -}}
+{{- end -}}
+
+
+{{/*
+Retrieve public key of minio certificate
+*/}}
+{{- define "minio.cert.publicKey" -}}
+{{- $values := (. | mustDeepCopy) -}}
+{{- $_ := set $values "commonCertOptions" (dict "certKeyName" $values.Values.certificate "publicKey" true) -}}
+{{ include "common.resources.cert" $values }}
+{{- end -}}
+
+
+{{/*
+Retrieve private key of minio certificate
+*/}}
+{{- define "minio.cert.privateKey" -}}
+{{- $values := (. | mustDeepCopy) -}}
+{{- $_ := set $values "commonCertOptions" (dict "certKeyName" $values.Values.certificate) -}}
+{{ include "common.resources.cert" $values }}
+{{- end -}}
```

Comment:
Can't this also be moved to the common chart? It's pretty likely to be used at a later stage for other applications...

Reply:
@Ornias1993 the volume bits could likely be there but i think it would be nice if i have another application where we do this and then move this together as i would have a bit more context on the usage.
The private/public key ones are basically still consuming from the library chart, i think it's fine for now.